src/pages/Home/components/Editor/components/NoteTextarea/NoteTextarea.js

```diff
@@ -26,7 +26,7 @@ const NoteTestarea = (props) => {
 
   return (
       <TextareaAutosize
-        spellcheck="false"
+        spellCheck="false"
```

Comment:
Thanks! That error was annoying. (There's a typo in the component name too, but I fixed that on my dev-branch)